submit_and_compare/submit_and_compare.py

```diff
@@ -9,13 +9,14 @@
 import pkg_resources
 
 from lxml import etree
-
 from django.template import Context, Template
 from django.utils.translation import ungettext
 
 from xblock.core import XBlock
 from xblock.fields import Scope, String, List, Float, Integer
 from xblock.fragment import Fragment
+from xblockutils.studio_editable import StudioEditableXBlockMixin
```

Comment:
I don't think this import actually gets used/is needed in the current form of this xblock.